contrib/fonttools/showttf.c

```diff
@@ -2887,6 +2887,7 @@ static void gdefshowligcaretlist(FILE *ttf,int offset,struct ttfinfo *info) {
 		printf( "!!!! Unknown caret format %d !!!!\n", format );
 	    }
 	}
+        free(offsets);
```

Comment:
The lines added in dumppfa.c also have mismatched leading whitespace.

I hate to nitpick, but people complain about unnormalized whitespace, and it's easy to fix it before it gets committed than afterwards.